synapse/storage/events.py

```diff
@@ -1548,6 +1556,86 @@ def _store_redaction(self, txn, event):
             (event.event_id, event.redacts),
         )
 
+    @defer.inlineCallbacks
+    def _censor_redactions(self):
+        """Censors all redactions older than a month that haven't been censored.
+
+        By censor we mean update the event_json table with the redacted event.
+
+        Returns:
+            Deferred
+        """
+
+        if not self.hs.config.redaction_retention_period:
```

Comment:
Not really important imho, but why add a handler that will be called every 10min only to return immediately if this flag isn't set rather than move `hs.get_clock().looping_call(_censor_redactions, 10 * 60 * 1000)` behind a `if self.hs.config.redaction_retention_period`?